.circleci/config.yml

```diff
@@ -16,7 +16,7 @@ jobs:
             echo "https://mirror.csclub.uwaterloo.ca/alpine/v3.10/community" >>/etc/apk/repositories
             apk update
             apk add bash curl git openssh
-            mkdir /code
+            mkdir /code -p
```

Comment:
why? the effect is same in this context.

Reply:
I made the changes to make the build work. Previously, working_directry was set to `/code`. Trying to make new directory `/code` in this step was giving an error, saying `directory already exists`. `-p` flag solves this.